lib/postgrex/type_module.ex

```diff
@@ -333,7 +333,7 @@ defmodule Postgrex.TypeModule do
   end
 
   defp decode_rows(dispatch, rest, acc, rem, full, rows) do
-    quote location: :keep do
+    quote location: :keep, generated: true do
```

Comment:
This one covers clauses like:
```
      defp decode_rows(<<?D, rest::binary>>, _, _, rows) do
        {:more, [?D | rest], rows, 0}
      end
```
in the code; is there a performance difference between creating an improper list which is immediately turned back into a binary and passing along the binary directly (maybe related to "small piece of a big buffer" issues?)
```
      defp decode_rows(<<?D, rest::binary>> = whole_thing, _, _, rows) do
        {:more, whole_thing, rows, 0}
      end
```